room_allocation.py

```diff
@@ -1,10 +1,13 @@
 import random
+import os
+from models import ModelRoom, ModelPerson, Base, create_engine, connect_db
 
 
 class Amity(object):
     def __init__(self):
         self.rooms = {}  # {'room_name':<obj: Room>}
```

Comment:
These comments not very descriptive. It will be better to just use plain english. Its really difficult to know that room name is an object of type Room the way you wrote.